korman/exporter/mesh.py

```diff
@@ -484,6 +616,37 @@ def _export_geometry(self, bo, mesh, materials, geospans, mat2span_LUT):
                         uvs.append(dPosDv)
                     geoVertex.uvs = uvs
 
+                    if export_deform:
+                        # Get bone ID and weight from the vertex' "group" data.
+                        # While we're at it, sort it by weight, and filter groups that
+                        # have no bone assigned. Take only the first 3 bones.
+                        weights = sorted([ \
+                                (group_id_to_bone_id[group.group], group.weight) \
+                                for group in source.groups \
+                                if group.weight > 0 \
+                            ], key=lambda t: t[1], reverse=True)
+                        if len(weights) > 3 and not warned_extra_bone_weights:
+                            warned_extra_bone_weights = True
+                            self._report.warn(f"'{bo.name}': only three bones can deform a vertex at a time. Please use Weight Tools -> Limit Total at 3 to ensure deformation is consistent between Blender and Plasma.")
```

Comment:
I wonder if there's a way we can do that programmatically? I seem to remember that PlasmaMax will pick the bones with the highest weights.